BeatSaverSharp/BeatSaver.cs

```diff
@@ -76,6 +76,61 @@ public BeatSaver(string applicationName, Version version) : this(new BeatSaverOp
             return await FetchBeatmap("maps/id/" + key, token).ConfigureAwait(false);
         }
 
+        public async Task<Dictionary<string, Beatmap>> BeatmapByHash(params string[] hash)
+        {
+            return await BeatmapByHash(default, false, hash);
+        }
+
+        public async Task<Dictionary<string, Beatmap>> BeatmapByHash(CancellationToken token, bool skipCacheCheck, params string[] hash)
```

Comment:
For consistency, I _think_ it would be better to not use the `params` keyword here and put the `hash` parameter first
(I'd also call it `hashes` to make it clear that there are possibly multiple hashes)